csFastFloat/csFastFloat.csproj

```
@@ -29,6 +29,10 @@
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
   </PropertyGroup>
 
+  <PropertyGroup Condition="'$(Configuration)|$(TargetFramework)|$(Platform)'=='Debug|net5.0|AnyCPU'">
+    <DocumentationFile>E:\TELUQ\Maitrise\csFastFloat\csFastFloat\csFastFloat.xml</DocumentationFile>
+  </PropertyGroup>
```

This generates the doc-file only for _Debug_. You'd like to have it only for release builds and keep the debug-build fast?
```suggestion
  <PropertyGroup Condition="'$(Configuration)' == 'Release'">
    <DocumentationFile>bin\Release\$(TargetFramework)\csFastFloat.xml</DocumentationFile>
  </PropertyGroup>
```
The MsBuild-property `$(...)` and the relative path elides the hard-coding of the path.